webapp/portlet/src/main/webapp/vue-apps/people-list/initComponents.js

```diff
@@ -4,6 +4,9 @@ import PeopleToolbar from './components/PeopleToolbar.vue';
 import PeopleCard from './components/PeopleCard.vue';
 import PeopleCardFront from './components/PeopleCardFront.vue';
 import PeopleCardReverse from './components/PeopleCardReverse.vue';
+import PeopleAdvancedFilterDrawer from './components/PeopleAdvancedFilterDrawer.vue';
+
+import * as ProfileSettingsService from '../common/js/ProfileSettingsService.js';
```

Comment:
I don't find this file, it's defined in https://github.com/Meeds-io/MIPs/issues/38 that isn't in "PR review" process yet, so I guess this MIP has to wait for Meeds-io/MIPs#38 to be merged isn't it ?

@rdenarie FYI

Reply:
Yes, this mips must be merged after mips#38, which will be in review in next hours.

We provide the pr so that you can start the review.

Reply:
yes already mentioned here, it is declared like that since in the other MIP it isn't yet a common component, should it should be updated there before.

Reply:
fixed